doc/readme/README.md.in

```diff
@@ -0,0 +1,141 @@
+Easy::jit: A just-in-time compiler for C++
+==========================================
+
```

Comment:
shit, I missed that one, it probably meas I should have updated this one...

Reply:
I removed that, but I moved all the headers required by the user to the same directory, to catch them with only one "-I" flag.